subsys/mpsl/cx/Kconfig

```diff
@@ -5,9 +5,7 @@
 #
 
 config MPSL_CX
-	bool "Radio Coexistence interface support [EXPERIMENTAL]"
-# Generic coexistence is not yet supported by Bluetooth
-	depends on !BT
```

Comment:
Probably this entry shall be moved to `config MPSL_CX_THREAD` and similar `depends on !NRF_802154_RADIO_DRIVER` should be introduced for `MPSL_CX_BT`

Reply:
Since it's been renamed to thread coexistence, I don't think we need to repeat this sentence. Agreed for the `depends on` though. I added `!BT` for thread coex too.

Reply:
Yeah, I meant `depends on !` entry. Not the comment above